src/verifier.spec.ts

```diff
@@ -75,16 +75,15 @@ describe('verifySDJWT', () => {
     expect(result).toEqual(expectedResult);
   });
 
-  it('should error when checking nonce and aud without a keybind', async () => {
+  xit('should error when checking nonce and aud without a keybind', async () => {
```

Comment:
@Linasi, why is this disabled? Was it done by mistake?

Reply:
Everything else looks good so approving it.

Reply:
> @linasi, why is this disabled? Was it done by mistake?

My bad. Forgot to enable it back after I finished checking something. Thanks for picking this up.